scripts/select_appointments.sql

```diff
@@ -1 +1,8 @@
 -- retrieve appointment information
+SELECT COUNT(*), (p_given_name, p_family_name, date, time),
+FROM appointments,
+WHERE d_given_name = 'Hubbard';
+
+SELECT COUNT(*), (d_given_name, d_family_name, date, time),
+FROM appointments,
+WHERE p_given_name = 'Wilcox';
```

Comment:
This would get the desired output:

```sql
-- retrieve appointment information
-- for Dr. Hubbard
SELECT p.family_name, a.date, a.time
  FROM patients p
  INNER JOIN appointments a
    ON a.patient_id = p.id
  INNER JOIN doctors d
    ON a.doctor_id = d.id
WHERE d.family_name = 'Hubbard';
-- for patient Wilcox
SELECT d.family_name, a.date, a.time
  FROM doctors d
  INNER JOIN appointments a
    ON a.doctor_id = d.id
  INNER JOIN patients p
    ON a.patient_id = p.id
WHERE p.family_name = 'Wilcox';
```